en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -77,10 +77,10 @@ DONT_RUN=1 make px4_sitl gazebo mavsdk_tests
 
 ### Run All PX4 Tests
 
-To run all tests, use:
+To run all SITL tests as defined in [sitl.json](https://github.com/PX4/Firmware/blob/master/test/mavsdk_tests/configs/sitl.json), do:
 
 ```sh
-test/mavsdk_tests/mavsdk_test_runner.py --speed-factor 20 --gui
+test/mavsdk_tests/mavsdk_test_runner.py test/mavsdk_tests/configs/sitl.json --speed-factor 10
```

Comment:
This looks great. Is `mavsdk_test_runner.py` documented anywhere? Ie. arguments? Would be good to add a note or similar explaining either what the options are, or noting that you can get them using (say) `mavsdk_test_runner.py -h`

Reply:
See just below. The arguments will change over time, I wouldn't document them in detail but just mention `-h`.